server/offenceCodeDecisions/DecisionTree.ts

```diff
@@ -24,43 +16,52 @@ export default new Decision()
       .title('What did the incident involve?')
       .child(
         decision('Assaulting someone')
-          .title('Who was assaulted?')
+          .title([
```

Comment:
We now have the decisions being able to be a bit different based on incident role - e.g. different titles